_summaries/yin2020center-based_2.md

```diff
@@ -1,19 +1,48 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{yin2020center-based}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: jchavezberkeley
+score: 10 # How did you like this paper 0(dislike) to 10(love)
 ---
-
-TODO: Summarize the paper:
 * What is the core idea?
+
+CenterPoint is a framework that represents, detects, and tracks 3D objects as points. It works first by detecting the centers of objects using keypoint detection and then estimates 3D size, 3D orientation, and velocity. CenterPoint has a second stage where it refines those estimates using additional point features. As described by the paper, CenterPoint simplifies 3D object tracking to greedy closest-point matching.
+
+CenterPoint's performance was state-of-the-art on the nuScenes benchmark for both 3D detection and tracking.
+
 * How is it realized (technically)?
+
+CenterPoint uses a Lidar-based backbone to build a representation of an input point-cloud. This helps extract map-view feature representation, and then, a standard image-based keypoint detector is used to find object centers.
+
+<img src="yin2020center-based_a.png" width="90%">
+
+During the first phase, the CenterPoint paper mentions 3 types of heads: center headmap heads, regression heads, velocity heads. The center-head's job is to produce a heatmap at the center location of any detected object. 
+
+The regression heads help provide "full state" information of 3D bounding boxes. Regression to log size also helps better handle boxes of different shapes. Then, at inference time, properties are extracted by indexing into the regression head outputs at each object's peak location.
+
+Two-dimensional velocity estimation is learned as another regression output. This is uses to predict the difference in object position between the current and past frame. 
+
+Finally, CenterPoint combines all the heatmap and regression losses and optimizes then before going onto the second phase.
+
+The second phase of CenterPoint extracts additional point-features from the backbone. Extracted point-features are passed through an MLP, and then, confidence scores and box refinements are predicted.
+
 * How well does the paper perform?
+
```

Comment:
How does this work improve on the previous work? What is unique about this methodology?